rskj-core/src/main/java/co/rsk/peg/lockingcap/LockingCapSupport.java

```diff
@@ -1,12 +1,13 @@
 package co.rsk.peg.lockingcap;
 
 import co.rsk.bitcoinj.core.Coin;
-import java.util.Optional;
 import org.ethereum.core.Transaction;
 
 public interface LockingCapSupport {
 
-    Optional<Coin> getLockingCap();
+    Coin getLockingCap();
```

Comment:
Didn't we talk about this? It returns empty pre RSKIP 134, doesn't it?

Reply:
You're right! Then, I'll apply this method for this Optional, `orElse(Optional.empty())`

Reply:
Done!